WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -19,11 +18,15 @@ namespace WalletWasabi.Fluent.ViewModels.Wallets;
 	Searchable = false)]
 public partial class WalletSettingsViewModel : RoutableViewModel
 {
+	private readonly IWalletModel _wallet;
 	[AutoNotify] private bool _preferPsbtWorkflow;
+	[AutoNotify] private string _walletName;
 
 	private WalletSettingsViewModel(IWalletModel wallet)
 	{
-		Title = $"{wallet.Name} - Wallet Settings";
+		_wallet = wallet;
+		Title = "Wallet Settings";
```

Comment:
Title is set via MetaData, you can remove this.

Reply:
Done